reconcile/jenkins/types.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import annotations
+
+import copy
+from enum import Enum
+from typing import Any
+
+from pydantic import BaseModel
+
+
+class SSHHostKeyVerificationStrategy(Enum):
+    MANUALLYTRUSTEDKEYVERIFICATIONSTRATEGY = "manuallyTrustedKeyVerificationStrategy"
+    MANUALLYPROVIDEDKEYVERIFICATIONSTRATEGY = "manuallyProvidedKeyVerificationStrategy"
+    NONVERIFYINGKEYVERIFICATIONSTRATEGY = "nonVerifyingKeyVerificationStrategy"
+    KNOWNHOSTSFILEKEYVERIFICATIONSTRATEGY = "knownHostsFileKeyVerificationStrategy"
+
+
+class SSHConnector(BaseModel):
+    credentialsId: str
+    port: int = 22
+    sshHostKeyVerificationStrategy: SSHHostKeyVerificationStrategy = (
+        SSHHostKeyVerificationStrategy.NONVERIFYINGKEYVERIFICATIONSTRATEGY
+    )
+
+    def to_dict(self) -> dict[str, Any]:
+        inst_vars = copy.deepcopy(vars(self))
+        inst_vars[
+            "sshHostKeyVerificationStrategy"
+        ] = self.sshHostKeyVerificationStrategy.value
+        return inst_vars
+
+
+class computerConnector(BaseModel):
+    sSHConnector: SSHConnector
+
+    def to_dict(self) -> dict[str, Any]:
+        return {"sSHConnector": self.sSHConnector.to_dict()}
+
+
+class JenkinsWorkerFleet(BaseModel):
```

Comment:
You could use the [field alias](https://docs.pydantic.dev/usage/schema/#field-customization) pedantic feature to use snake_case variable names, e.g.,

```
min_spare_size: int = Field(0, alias="minSpareSize")
```